src/elements/content-explorer/stories/RenameDialog.stories.js

```diff
@@ -0,0 +1,53 @@
+// @flow
+import * as React from 'react';
+import { useArgs } from '@storybook/preview-api';
+
+import PrimaryButton from '../../../components/primary-button/PrimaryButton';
```

Comment:
can we replace this with a blueprint button?

Reply:
sure! sounds good!